src/core/utils.js

```diff
@@ -211,7 +211,7 @@ export function normalizePadding(text = "") {
   }
   doc.normalize();
   // use the first space as an indicator of how much to chop off the front
-  const firstSpace = doc.body.innerText
+  const firstSpace = doc.body.textContent
```

Comment:
I think we should keep this as it is... Or we just replace this bulky function with `reindent()` in `core/reindent`.

Reply:
I'd like to eventually use `reindent()` - that would be great. I need to try it and make sure it handles all the special cases that this one needed to deal with.

Reply:
In any case, I don't think there is any valid reason for this to be using `innerText`

Reply:
I was concerned about `innerText`s special behavior that converts `<br>` to `\n` and we are using the exact character here. Thinking more, yeah, I think it's okay. This is currently only for markdown and IDL strings that won't (normally) use any `<br>`.

Reply:
Understood. I didn't think of that, but I agree that it should be ok. The markdown is supposed to be processed as given, not as displayed.